src/core/bot/interact.py

```
@@ -237,3 +197,21 @@ def reply_qrcode(message: RobotMessage):
     qr_img.save(f"{cached_prefix}.png")
 
     message.reply("生成了一个二维码", png2jpg(f"{cached_prefix}.png"))
+
+
+@command(tokens=["晚安", "睡觉", "睡觉去了", "sleep"], is_command=False)
+def reply_mc_sleep(message: RobotMessage):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)